src/pages/Contributing/Glossary/index.md

```diff
@@ -1,4 +1,97 @@
+
+## Bugfix
+
+A bug fix is a change to a system or a product designed to handle a code or a design bug/glitch.
+
+## Bug report
+
+A reported problem of any part of the design system (sketch files, style guide, code, documentation, usage, specs…).
+
+## Repository
+
+A repository (usually called “repo”) is a location where you will find all the files for a particular project.
+
+## Git
+
+Git is an open-source version control system that stores the modifications in a central repository.
+GitHub
+GitHub is a project hosting platform for collaboration and version control. It allows people to work together on a product.
+
+## Abstract
+
+Abstract is a software like GitHub, using by the designers to collaborate on design files.
+
+## Master
+
+This is the source of truth of our work where you will find the latest and safest version of Mozaic.
+
+## Branch
+
+Branching, in version control and software configuration management, is the duplication of an object under version control so that modifications can occur in parallel along multiple branches. Learn more (https://help.github.com/en/articles/about-branches)
+
+## Pull request
+
+Pull requests let you tell others about changes you've pushed to a GitHub repository. Once a pull request is sent, interested parties can review the set of changes, discuss potential modifications, and even push follow-up commits if necessary.
+
+## Markdown
+
+A simple way for non-programming types to write in an easy-to-read format that could be converted directly into HTML. Learn more (https://en.wikipedia.org/wiki/Markdown#Example)
+
+## Documentation
+
+Documentation is a crucial part of the product. It allows everyone to explain their work, their uses cases, and their conventions. Good documentation will be able to onboard new users in an easier way.
+
+## Tokens
+
+Design tokens are UI pieces of information that store visual design attributes, such as values for margins and spacing, font sizes and families, or color values.
+Lean more (https://www.youtube.com/watch?v=wDBEc3dJJV8)
+
+## Specs
+
+All the important pieces of information to compose our designs in different technology. So if you want to create the patterns in a different technology, you have all the properties behind font size, border-radius, heigh, padding, and all single aspects inside a pattern.
+
+## Magic Unit
+
+A magic unit is a base unit for any element or property that require a defined dimension. It is equal to 16px but is expressed in rem. We multiply or divide it to define white spaces, icons sizes, line-heights etc…
+
+## Atomic design
+
+The Atomic design is a new modular design methodology created by Brad Frost, helpful to create design systems. It consists to start from the simple elements, and reusable it to create more complex components. It permits to the designers to create the most consistent user interfaces.
```

Comment:
Atomic Design is a new modular design methodology created by Brad Frost helpful to create design systems. It consists to start from the simple elements, and reusable it to create more complex components. It allows the designers to create the most consistent user interfaces. This is a methodology we use when we design for Mozaic.